node/packages/aws-lambda-otel-extension/internal/otel-extension-internal-node/index.js

```diff
@@ -201,7 +201,7 @@ const responseHandler = async (span, { res, err }, isTimeout) => {
     functionData.errorExceptionType = 'TimeoutError';
     functionData.errorExceptionMessage = '';
     functionData.errorExceptionStacktrace =
-      'Function execution is going to exceeded configured timeout limit.';
+      'Function execution exceeded the configured timeout limit.';
```

Comment:
To be perfectly clear, we don't know that yet, at this point. It's still possible that the function will manage to succeed on time.

I think that the right approach here is to not report `errorExceptionStacktrace` for timeouts, as it's clear we can't have a stack trace here, so this data is N/A

Also, currently our timeout reporting is unreliable, I've described the issue in more detail in `SC-263`